rules/report-bugs-and-suggestions/rule.md

```diff
@@ -26,14 +26,20 @@ When reporting bugs and giving product feedback, it is essential that you are as
 
 Here are the 8 tips:
 
-* [Tip #1: Draft your bug with enough details ](#tip-1-draft-your-bug-with-enough-details)   
-* [Tip #2: Draft your suggestion with the complaint and what you expect to see](#tip-2-draft-your-suggestion-with-the-complaint-and-what-you-expect-to-see)
-* [Tip #3: Should you send this to the Product Owner or the Tech Lead?](#tip-3-should-you-send-this-to-the-product-owner-or-the-tech-lead)  
-* [Tip #4: Should you email or put it in the backlog?](#tip-4-should-you-email-or-put-it-in-the-backlog)  
-* [Tip #5: Do you make it easy to find all the backlog in your company?](#tip-5-do-you-make-it-easy-to-find-all-the-backlog-in-your-company)  
-* [Tip #6: Make sure when using backlog, the Product Owner will still get an email](#tip-6-make-sure-when-using-backlog-the-product-owner-will-still-get-an-email)  
-* [Tip #7: Separate PBIs](#tip-7-separate-pbis)  
-* [Tip #8: Use emojis and prefixes for PBI/Issues titles, or email subjects](#tip-8-use-emojis-and-prefixes-for-pbiissues-titles-or-email-subjects)  
+- [Tip #1: Draft your bug with enough details](#tip-1-draft-your-bug-with-enough-details)
+- [Hi team,](#hi-team)
+- [Hi team,](#hi-team-1)
+- [Hi Danny,](#hi-danny)
+- [Hi Danny,](#hi-danny-1)
+- [Expected result](#expected-result)
+- [Actual result](#actual-result)
```

Comment:
What happened here? Is this intentional?

Reply:
Thanks for catching... it's not... things went weird
 
![JudgingYouGIF](https://user-images.githubusercontent.com/5680199/222431746-26222d4a-7056-4f7a-88a8-8b0952f67ee4.gif)

Reply:
The markdown all in one extension has a auto table of contents which made the formatting weird

![image](https://user-images.githubusercontent.com/5680199/222435965-efc6f491-a7cc-4070-8029-e61e57ce42da.png)